DownloadManager/Plugin.DownloadManager.Android/DownloadFileImplementation.cs

```diff
@@ -92,20 +94,44 @@ public DownloadFileImplementation (ICursor cursor)
             Url = cursor.GetString (cursor.GetColumnIndex (Android.App.DownloadManager.ColumnUri));
         }
 
-        public void StartDownload (Android.App.DownloadManager downloadManager, string destinationPathName)
+        private bool _mobileNetworkAllowed = true;
+
+        public bool MobileNetworkAllowed
         {
-            using (var downloadUrl = Uri.Parse (Url))
-            using (var request = new Android.App.DownloadManager.Request (downloadUrl)) {
+            get
+            {
+                return _mobileNetworkAllowed;
+            }
+            set
+            {
+                _mobileNetworkAllowed = value;
+                if (Request != null && value == false) {
+                    Request.SetAllowedNetworkTypes(Android.App.DownloadNetwork.Wifi);
```

Comment:
The original reason for not using setAllowedOverMetered was the the Android Target framework was set to API level 15, which did not include this method. I see however from this graph that only 1.4% of all Android devices run on Ice Cream Sandwich. I will therefore increase the target framework to Android 4.1.